_layouts/_site/default.html

```diff
@@ -7,7 +7,7 @@
   <head>
     <meta charset="utf-8">
     <title>{{ page.title }}</title>
-    <link rel="stylesheet" href="/assets/css/styles.css">
+    <link rel="stylesheet" href="assets/main.css">

```

Comment:
Did you change this manually, or did Jekyll change these?

Reply:
This was changed manually, /assets/css/styles.css doesn't exist, but assets/main.css does, so I changed it to that.